packages/components/src/Tooltip/src/PassiveTrigger.tsx

```diff
@@ -0,0 +1,80 @@
+import { useStyledSystem, type StyledSystemProps } from "@hopper-ui/styled-system";
+import clsx from "clsx";
+import { forwardRef, useRef, type ForwardedRef, type ReactNode } from "react";
+import { useFocusable } from "react-aria";
+import { useContextProps } from "react-aria-components";
+
+import { cssModule, type BaseComponentDOMProps } from "../../utils/index.ts";
+
+import { PassiveTriggerContext } from "./PassiveTriggerContext.ts";
+
+import styles from "./PassiveTrigger.module.css";
+
+export const GlobalPassiveTriggerCssSelector = "hop-PassiveTrigger";
+
+export interface PassiveTriggerProps extends StyledSystemProps, BaseComponentDOMProps {
+    /**
+     * The children of the PassiveTrigger.
+     */
+    children?: ReactNode;
+}
+/**
+ * A PassiveTrigger wraps a trigger element and Tooltip, handling visibility and positioning.
+ *
+ * [View Documentation](TODO)
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
is the comment still necessary?  and does it prevent the jsdoc from applying on the PassiveTrigger?

Reply:
removed